packages/nextjs/src/config/wrappers/wrapperUtils.ts

```diff
@@ -83,8 +84,8 @@ export function callTracedServerSideDataFetcher<F extends (...args: any[]) => Pr
       // TODO: Extract trace data from `req` object (trace and baggage headers) and attach it to transaction
 
       const newTransaction = startTransaction({
-        op: 'nextjs.data',
-        name: options.parameterizedRoute,
+        op: 'nextjs.data.server',
```

Comment:
Changed to `nextjs.data.server` from `nextjs.data` so we can differenciate data fetching methods that are run on the server vs. those that are run on the client (ie `nextjs.data.client`).
`getInitialProps` will run on the client, when navigating via `next/link` or `next/router`.